liblwgeom/gbox.c

```diff
@@ -723,3 +724,103 @@ void gbox_float_round(GBOX *gbox)
 	}
 }
 
+
+
+/* Unfortunately including advanced instructions is something that
+only helps a small sliver of users who can build their own
+knowing the target system they will be running on. Packagers
+have to aim for the lowest common denominator. So this is
+dead code for the forseeable future. */
+#define HAVE_PDEP 0
```

Comment:
Yeah, you're right, it needs to go. Any kind of cpu-specific instruction is kind of out-of-bounds for us, because you cannot control what the build machine has vs what the deploy machine has. They end up being only useful for that sliver of users that build their own binaries for their target architectures.